components/calendar/src/lib.rs

```diff
@@ -116,6 +116,7 @@ mod date;
 mod datetime;
 
 pub mod any_calendar;
+pub mod astronomy;
```

Comment:
issue: ought this be public? I would have imagined this is crate-internal for calculations

Reply:
I changed this to not be public